mentirinha/counter.py

```diff
@@ -0,0 +1,16 @@
+import redis
+
+
+redis_client = redis.Redis(**settings.REDIS_CONNECTION_PARAMS)
+
+
+def incr(url):
+    return redis_client.incr(f'url_{url}')
```

Comment:
Esse prefixo tem que ser configurável.

Reply:
ok, vou deixar configuravel, mas tem 16 bancos também para usar (se a ideia de deixar configuravel for para evitar conflito)